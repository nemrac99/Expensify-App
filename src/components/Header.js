import React from 'react';
import { Link, NavLink } from "react-router-dom";

const Header = () => (
	<header>
		<h1>Expensify</h1>
		<div>
			<NavLink to="/" activeClassName="isActive" exact={true}>Dashboard</NavLink>
			<NavLink to="/create" activeClassName="isActive">Create</NavLink>
			<NavLink to="/help" activeClassName="isActive">Help</NavLink>
		</div>
	</header>
);

export default Header;