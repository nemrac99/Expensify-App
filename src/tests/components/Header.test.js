import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import Header from "../../components/Header";
test("render header", () => {
	const wrapper = shallow(<Header />)
	expect(toJSON(wrapper)).toMatchSnapshot();
});