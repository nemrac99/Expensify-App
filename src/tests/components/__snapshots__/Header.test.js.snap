// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <div>
    <NavLink
      activeClassName="isActive"
      aria-current="page"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>
    <NavLink
      activeClassName="isActive"
      aria-current="page"
      to="/create"
    >
      Create
    </NavLink>
    <NavLink
      activeClassName="isActive"
      aria-current="page"
      to="/help"
    >
      Help
    </NavLink>
  </div>
</header>
`;
